lib/msf/util/exe.rb

```diff
@@ -21,14 +21,17 @@ class EXE
 require 'msf/core/exe/segment_injector'
 require 'msf/core/exe/segment_appender'
 
-  ##
+  # self.set_template_default
+  # Generates a default template
   #
-  # Helper functions common to multiple generators
-  #
-  ##
-
+  # @param  opts [Hash] The options hash
```

Comment:
yard automatically makes a `@param` for option hashes if you have any `@option` tags, so you don't need this at all.